executor/join_test.go

```diff
@@ -1356,7 +1357,7 @@ func TestIndexLookupJoin(t *testing.T) {
 	tk.MustExec("analyze table s;")
 
 	tk.MustQuery("desc format = 'brief' select /*+ TIDB_INLJ(s) */ count(*) from t join s use index(idx) on s.a = t.a and s.b < t.b").Check(testkit.Rows(
-		"HashAgg 1.00 root  funcs:count(1)->Column#6",
```

Comment:
Expected since model2 prefers to use StreamAgg instead of HashAgg if no much data to process.